deployments/deployments.json

```diff
@@ -95,6 +96,10 @@
     "MulticallHandler": { "address": "0x863859ef502F0Ee9676626ED5B418037252eFeb2", "blockNumber": 36906393 }
   },
   "420": { "SpokePool": { "address": "0xeF684C38F94F48775959ECf2012D7E864ffb9dd4", "blockNumber": 17025501 } },
+  "690": {
+    "SpokePool": { "address": "0x28077B47Cd03326De7838926A63699849DD4fa87", "blockNumber": 5158526 },
+    "MulticallHandler": { "address": "0x924a9f036260DdD5808007E1AA95f08eD08aA569", "blockNumber": 5159031 }
+  },
```

Comment:
SpokePoolVerifier address is currently missing from here due to difficulties in deploying to the same address. Will be fixed shortly.

Reply:
Can we add the address here now?

Reply:
Hm sorry; I thought I'd added it. Working on that now.

Reply:
https://github.com/across-protocol/contracts/pull/570/commits/9f901d18aa5a6f0ce921ee9c4a1ccd7ba282dbd3